pkgs/games/dwarf-fortress/default.nix

```diff
@@ -35,6 +47,13 @@ let
   dfHackWorksWithCurrentVersion = true;
   dfHackEnabled = dfHackWorksWithCurrentVersion && enableDFHack;
 
+  themesThatWorkWithCurrentVersion = [
```

Comment:
I assume you read my explanation here after you asked about the same part of the code below... If you still feel that it's not a good idea after this explanation even, I'll drop it. I wrote it since, as a user, I'd rather not be able to install something that doesn't work. That would be surprising to me.